.ci/matrix_job.yaml

```diff
@@ -184,10 +186,11 @@ steps:
   - name: Style
     enable: ${do_style}
     containerSelector:
-      - "{name: 'toolbox', category: 'tool'}"
+      - "{name: 'style', category: 'tool'}"
     agentSelector:
       - "{nodeLabel: 'skip-agent'}"
     run: |
+      export COMPILE_DPCP=false
```

Comment:
i prefer not to do such change now if its not needed (even if current behavior is wrong) can we remove it and keep build dpcp ?